### PR TITLE
update istio and remove kfserving

### PR DESCRIPTION
**Pull request by ryansteakley**
**Description of your changes:**

- Upstream is using istio 1-14 and kfserving has been removed
- Verified that cognito/rds-s3/sanity-check and manifests installation all pass

**Testing:**
- [ ] Unit tests pass
- [ ] e2e tests pass
- Details about new tests (If this PR adds a new feature)
- Details about any manual tests performed

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by mbaijal — COMMENTED** [review]
Changes look ok but could you please add some details on tests run.

**Comment by ryansteakley** [comment]
> Changes look ok but could you please add some details on tests run.

Didn't save the test logs, but have verified that they are passing. This pr in particular is mostly concerned with getting our kustomization files to point to the correct paths upstream.

**Review by surajkota — CHANGES_REQUESTED** [review]
Also change the version in main readme badge, test constant, prerequisites to point to the right upstream versions

**Review by mbaijal — APPROVED** [review]

**Comment by ryansteakley** [comment]
> Also change the version in main readme badge, test constant, prerequisites to point to the right upstream versions

updatd the test constants. Not sure what you mean by main readme badge. Waiting on pointing pre-reqs to upstream since they are still tagged as rc version and we don't have an aws release either.

File: tests/e2e/pa.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import boto3
```
Comment (surajkota) [inline tests/e2e/pa.py:1]: is it here by mistake?

File: website/content/en/docs/component-guides/kserve.md [hunk 2]
```diff
@@ -8,6 +8,8 @@ This tutorial shows how to set up a load balancer endpoint for serving predictio
 
 > Note: Kubeflow on AWS v1.4 uses [KFServing](https://www.kubeflow.org/docs/external-add-ons/kserve/kserve/#kfserving-is-now-kservehttpskservegithubiowebsite07blogarticles2021-09-27-kfserving-transition). The KFServing project is now called KServe.
```
Comment (surajkota) [inline website/content/en/docs/component-guides/kserve.md:9]: lets remove both the lines or maybe just add one line about KFServing project is now called KServe wiht this link https://kserve.github.io/website/0.7/blog/articles/2021-09-27-kfserving-transition/#kfserving-is-now-kserve and link to this guide for migration https://kserve.github.io/website/0.7/admin/migration/

**Review by surajkota — CHANGES_REQUESTED** [review]

File: tests/e2e/pa.py (hunk 1 above)
Comment (ryansteakley) [reply tests/e2e/pa.py:1]: yes will remove it

File: website/content/en/docs/component-guides/kserve.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ weight = 30
 
 This tutorial shows how to set up a load balancer endpoint for serving prediction requests over an external DNS on AWS.
 
-> Note: Kubeflow on AWS v1.4 uses [KFServing](https://www.kubeflow.org/docs/external-add-ons/kserve/kserve/#kfserving-is-now-kservehttpskservegithubiowebsite07blogarticles2021-09-27-kfserving-transition). The KFServing project is now called KServe.
+The KFServing project is now called [KServe](https://kserve.github.io/website/0.7/blog/articles/2021-09-27-kfserving-transition/#kfserving-is-now-kserve). To migrate from KFServing to Kserve follow this [guide](https://kserve.github.io/website/0.7/admin/migration/).
```
Comment (surajkota) [inline website/content/en/docs/component-guides/kserve.md:9]: nit: add  `> Note:` before the line

**Review by surajkota — APPROVED** [review]

File: website/content/en/docs/component-guides/kserve.md (hunk 3 above)
Comment (ryansteakley) [reply website/content/en/docs/component-guides/kserve.md]: done

**Pull request merged**